packages/host/app/services/matrix-service.ts

```diff
@@ -435,7 +435,7 @@ export default class MatrixService extends Service {
   ) {
     displayName = displayName.trim();
     this._isInitializingNewUser = true;
-    this.start({ auth });
+    await this.start({ auth, skipRealmAuthentication: true });
```

Comment:
Is it possible to somehow get the the user id before calling `this.start()`? That way we could create the user before `start`ing.

Reply:
I do have the Matrix username:

<img width="554" height="112" alt="Image" src="https://github.com/user-attachments/assets/f3281c60-6d81-4084-afe7-89fb729041aa" />

but calling `createUser` fails because `this.start` hasn’t been called. I don’t understand how to get out of this chicken-and-egg situation without something like `skipRealmAuthentication`